guacamole-existing-vnet/azuredeploy.json

```diff
@@ -0,0 +1,236 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
```

Comment:
Makes sense in this case. Btw will you be extending this template to make it existing or new VNET?